src/test/java/teammates/storage/sqlapi/FeedbackResponsesDbTest.java

```diff
@@ -0,0 +1,130 @@
+package teammates.storage.sqlapi;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+
+import java.util.UUID;
+
+import org.mockito.MockedStatic;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.FeedbackResponse;
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@code FeedbackResponsesDb}.
+ */
+public class FeedbackResponsesDbTest extends BaseTestCase {
+
+    private FeedbackResponsesDb feedbackResponsesDb;
+
+    private MockedStatic<HibernateUtil> mockHibernateUtil;
+
+    @BeforeMethod
+    public void setUpMethod() {
+        mockHibernateUtil = mockStatic(HibernateUtil.class);
+        feedbackResponsesDb = spy(FeedbackResponsesDb.class);
+    }
+
+    @AfterMethod
+    public void tearDownMethod() {
+        mockHibernateUtil.close();
+    }
+
+    @Test
+    public void testCreateFeedbackResponse_feedbackResponseDoesNotExist_success()
+            throws InvalidParametersException, EntityAlreadyExistsException {
+        FeedbackResponse feedbackResponse = getTypicalFeedbackResponse();
+
+        feedbackResponsesDb.createFeedbackResponse(feedbackResponse);
+
+        mockHibernateUtil.verify(() -> HibernateUtil.persist(feedbackResponse));
```

Comment:
Is there a reason for not also adding the `times(1)`? Same question for `testUpdateFeedbackResponse_feedbackResponseExists_success()` and `testDeleteFeedbackResponse_feedbackResponseExists_success()` below - not sure if I missed something since other methods use that